change-notes/1.22/analysis-cpp.md

```diff
@@ -22,3 +22,4 @@
 - The predicate `TypeMention.toString()` has been simplified to always return the string "`type mention`".  This may improve performance when using `Element.toString()` or its descendants.
 - The `semmle.code.cpp.security.TaintTracking` library now considers a pointer difference calculation as blocking taint flow.
 - Fixed the `LocalScopeVariableReachability.qll` library's handling of loops with an entry condition is both always true upon first entry, and where there is more than one control flow path through the loop condition.  This change increases the accuracy of the `LocalScopeVariableReachability.qll` library and queries which depend on it.
```

Comment:
Not in this PR, but there's something wrong here:
"with an entry condition is both always true upon first entry"
This isn't grammatical. Perhaps this is meant to say:
"where an entry condition is both always true upon first entry"

Reply:
Being very nit-picky "queries which depend on it" should also be "queries that depend on it"